.changelog/3018.bugfix.md

```diff
@@ -0,0 +1,9 @@
+go/worker/sentry: Do client authentication on sentry control grpc
+
+The following configuration changes are needed due to this change:
```

Comment:
Should we move this to a `cfg` fragment so all config changes are in the same place?

Reply:
yeah maybe better that way

Reply:
i just moved everything to the `cfg` fragment, since otherwise the config change wouldn't have any context?

Not sure, could also keep the "bugfix" description in a separate fragment and keep the `cfg` fragments only with list of changes?

Reply:
> Not sure what we want to do in general, could also have the fix/feature descriptions in separate fragments and have the cfg fragments only list the changes?

Yeah, I think this makes the most sense.